src/panels.cpp

```diff
@@ -847,7 +847,7 @@ static void draw_stat_wide( const draw_args &args )
     std::pair<std::string, nc_color> activity = display::activity_text_color( u );
 
     static const std::string weary_label = translate_marker( "Weary:" );
-    static const std::string activity_label = translate_marker( "Activ:" );
```

Comment:
Not sure about this at all but this may be intended due to limited space of characters available in that column.
There is another case of that above.

@anothersimulacrum appears to have added that, so they may provide some insight.

Reply:
I mistook this one for the `active` activity string